v2/samples/direct_method_invocation_sample/AnalyzerModule/.gitignore

```diff
@@ -0,0 +1,31 @@
+# .NET Core
```

Comment:
Can you rename the project to EngineAnalyzer so it is consist with the docker image name?

Reply:
It's a personal preference, but I like to have the module "key" be the same as the docker image name and the routes match as well:

![image](https://user-images.githubusercontent.com/2163001/36084621-d892c1a2-0f73-11e8-8fec-c2b76f285359.png)